cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,298 @@
+"""
+This tool allows you to partially clone a data from courtlistener.com to your
+local environment, you only need to pass the type and object id and run it.
+
+e.g.
+
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Opinion --id 9355884
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Docket --id 5377675
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Person --id 16207
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Court --id usnmcmilrev
+
+
+This tool is only for development purposes, to enable it you need to set
+environment variable DEVELOPMENT to True. Also need to set CL_API_TOKEN env
+variable.
+
+This is still work in progress, some data is not cloned yet.
+
+"""
+
+import os
+import sys
+
+import environ
+import requests
+from django.db import transaction
+from django.utils.dateparse import parse_date
+
+from cl.lib.command_utils import VerboseCommand
+from cl.people_db.models import Person
+from cl.search.models import Citation, Court, Docket, Opinion, OpinionCluster
+from cl.search.tasks import add_items_to_solr
+
+env = environ.FileAwareEnv()
+DEVELOPMENT = env.bool("DEVELOPMENT", default=False)
+
+VALID_TYPES = ("Opinion", "Docket", "Person", "Court")
+
+cluster_endpoint = "https://www.courtlistener.com/api/rest/v3/clusters/"
+people_endpoint = "https://www.courtlistener.com/api/rest/v3/people/"
+courts_endpoint = "https://www.courtlistener.com/api/rest/v3/courts/"
+
+
+class Command(VerboseCommand):
+    help = (
+        "A helper function clone a data from courtlistener.com into local "
+        "environment "
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.verbosity = None
+        self.options = []
+        self.type = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--type",
+            type=str,
+            choices=VALID_TYPES,
+            help="Object type to clone. Current choices are %s"
+            % ", ".join(VALID_TYPES),
+        )
+
+        parser.add_argument(
```

Comment:
Could ID take multiple values so that you can clone tens or hundreds of items with something like:

```
clone_from_id --type foo --ids abc def foo bar
```